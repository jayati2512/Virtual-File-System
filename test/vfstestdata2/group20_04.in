createvfs demo4 500
mountvfs demo4 
makedir /home d
makedir /home a
makedir /home/d demo
listdir /home 1 /tmp/outputdir 
movedir /home/d/demo /home/a
makedir /home/a demo
listdir /home 1 /tmp/outputdir 
deletedir /home/a/demo
listdir /home 1 /tmp/outputdir 
makedir /home/sub1 sub2
addfile /home/sub1 demo.txt /usr/include/stdio.h
searchfile demo.txt /tmp/outputdir
listdir /home 1 /tmp/outputdir
copyfile /home/sub1/demo.txt /home/demo.txt
listdir /home 1 /tmp/outputdir
listfile /home/demo.txt viki.txt
unmountvfs demo4
createvfs demo4 100
mountvfs demo4 
searchfile demo.txt /tmp/outputdir
removefile /home/sub1/demo.txt
listdir /home 1 /tmp/outputdir
listfile /home/demo.txt fout
updatefile /home/demo.txt /usr/include/memory.h
listfile /home/demo.txt fout2
removefile /home/demo.txt
searchfile demo /tmp/outputdir
makedir /a/b/c/d/e ab
makedir /a/b/c f
listdir / 1 /tmp/outputdir
deletedir /a/b/c/d
listdir / 1 /tmp/outputdir
movedir /a/b /
deletedir /a
listdir / 1 /tmp/outputdir
unmountvfs demo4 
createvfs demo4 500
mountvfs demo4 
makedir /home d
makedir /home a
makedir /home/d demo
listdir /home 1 /tmp/outputdir 
movedir /home/d/demo /home/a
makedir /home/a demo
listdir /home 1 /tmp/outputdir 
deletedir /home/a/demo
listdir /home 1 /tmp/outputdir 
makedir /home/sub1 sub2
addfile /home/sub1 demo.txt /usr/include/stdio.h
searchfile demo.txt /tmp/outputdir
listdir /home 1 /tmp/outputdir
copyfile /home/sub1/demo.txt /home/demo.txt
listdir /home 1 /tmp/outputdir
listfile /home/demo.txt viki.txt
unmountvfs demo4
createvfs demo4 100
mountvfs demo4 
searchfile demo.txt /tmp/outputdir
removefile /home/sub1/demo.txt
listdir /home 1 /tmp/outputdir
listfile /home/demo.txt fout
updatefile /home/demo.txt /usr/include/memory.h
listfile /home/demo.txt fout2
removefile /home/demo.txt
searchfile demo /tmp/outputdir
makedir /a/b/c/d/e ab
makedir /a/b/c f
listdir / 1 /tmp/outputdir
deletedir /a/b/c/d
listdir / 1 /tmp/outputdir
movedir /a/b /
deletedir /a
listdir / 1 /tmp/outputdir
unmountvfs demo4 
