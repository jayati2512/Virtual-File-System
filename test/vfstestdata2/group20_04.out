createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
listdir_SUCCESS
movedir_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
listdir_SUCCESS
deletedir_SUCCESS
listdir_SUCCESS
makedir_SUCCESS
addfile_SUCCESS
searchfile_SUCCESS
listdir_SUCCESS
copyfile_SUCCESS
listdir_SUCCESS
listfile_SUCCESS
unmountvfs_SUCCESS
createvfs_FAILURE DATA_FILE_ALREADY_EXISTS
mountvfs_SUCCESS
searchfile_SUCCESS
removefile_SUCCESS
listdir_SUCCESS
listfile_SUCCESS
updatefile_SUCCESS
listfile_SUCCESS
removefile_SUCCESS
searchfile_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
listdir_SUCCESS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
listdir_SUCCESS
movedir_SUCCESS
deletedir_SUCCESS
listdir_SUCCESS
unmountvfs_SUCCESS
createvfs_FAILURE DATA_FILE_ALREADY_EXISTS
mountvfs_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
makedir_SUCCESS
listdir_SUCCESS
movedir_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
listdir_SUCCESS
deletedir_SUCCESS
listdir_SUCCESS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
addfile_SUCCESS
searchfile_SUCCESS
listdir_SUCCESS
copyfile_SUCCESS
listdir_SUCCESS
listfile_SUCCESS
unmountvfs_SUCCESS
createvfs_FAILURE DATA_FILE_ALREADY_EXISTS
mountvfs_SUCCESS
searchfile_SUCCESS
removefile_SUCCESS
listdir_SUCCESS
listfile_SUCCESS
updatefile_SUCCESS
listfile_SUCCESS
removefile_SUCCESS
searchfile_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
listdir_SUCCESS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
listdir_SUCCESS
movedir_FAILURE DESTINATION_ALREADY_HAVE_SOURCE_DIR
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
listdir_SUCCESS
unmountvfs_SUCCESS
